README.md

```diff
@@ -1 +1,96 @@
-# k8ssandra-terraform
\ No newline at end of file
+## Standerdization 
+
+terraform Style Guide to follow
+https://github.com/jonbrouse/terraform-style-guide/blob/master/README.md#resource-naming
+
+Linting and Verifications standards to be checked with the following commands depending on the file type:
+
+### Terraform
```

Comment:
We need a "#" header that is the project name, and then an intro to what the project is.